core-bundle/src/Routing/ResponseContext/HtmlHeadBag/HtmlHeadBag.php

```diff
@@ -26,6 +26,11 @@ final class HtmlHeadBag
 
     private array $keepParamsForCanonical = [];
 
+    /**
+     * @var array<string, string>
+     */
+    private array $metaTags = [];
```

Comment:
There are different types of `<meta>` tags, do we only want to support `<meta name content>`?
How about things like `<meta property="og:image" content="…">`?

We could make the meta tags just be an array of `HtmlAttributes` objects to support all types.
`$headBag->addMetaTag((new HtmlAttributes())->set('property', 'og:image')->set('content', '…'))`

Reply:
I agree, if needed we can still add convenience methods.

Reply:
> Agreed - but that also means we can't really remove specific meta tags?

We could add removal based on attribute and value like `->removeMetaTag('name', 'description')` and `->removeMetaTag('property', 'og:image')`

Reply:
In any case, see a711505a44